server/src/assets/game/driver-game.html

```diff
@@ -310,6 +312,9 @@ <h5 id="serverModalTitle" class="modal-title"></h5>
 			</div>
 		</div>
 	</form>
+	<div class="sessionTimer fixed-button">
+		<span id="timer"><span id="timerMinute">00</span>:<span id="timerSecond">00</span></span>
+	</div>
```

Comment:
What does this actually look like during game play?
Do players know what it's for?

Reply:
![image](https://user-images.githubusercontent.com/48696584/236448592-b048332d-4620-40e6-ad78-f88c44facf77.png)

There would be the possiblity to add a tag / description before the timer

Reply:
I'd say the looks aren't done. There's too little spacing to the elements above. A description/tag in front of the timer might also be needed - it could say something like "Fahrzeit" or similar in German. Not sure about the english version.

Reply:
Also, consistency with use of colons ("Ziel:" vs. "Geschwindigkeit"). IMO it should be all or none.